README.md

```diff
@@ -752,7 +798,13 @@ The `SimpleJmx` trait allows mixing in a simple JMX server. The JMX server is co
 
 ## Spray
```

Comment:
I'm a bit confused about c8adfcd3c0b962b58ca001a6daf23cc4e9f06e81

Spray has been superseded by akka-http, and that module only provides akka-http directives, so I think that we should keep the akka-http name and update the README instead.

Reply:
You're right. I had started with `akka-http` and when got to the README panicked and changed to `spray` 😅 Changed back the module name to `akka-http` and updated the README on 377f730fcc3c61d43f8840ce0911d97d6aa4a619.